### PR TITLE
Number format

### LINKED ISSUE #1493: The angular local don't looks to be initialysed correctly ...

### PR DESCRIPTION
**Pull request by sbrunner**
fix #1493, except #1553

### CODE REVIEW

File: src/services/filters.js [hunk 1]
```diff
@@ -218,10 +218,6 @@ ngeo.NumberCoordinates = function($filter) {
    *     Where "{x}" will be replaced by the first coordinate and "{y}" by the
    *     second one. Note: Use a html entity to use the semicolon symbole
    *     into a template.
-   * @param {(boolean|string)=} opt_localize Optional. If true or not defined,
-   *     format number as the current local system (see Angular number filter).
-   *     Set it explicitely to false to use always "." as the decimal separator
-   *     and include "'" group separators after each third digit.
    * @return {string} Number formated coordinates.
    */
   var filterFn = function(coordinates, opt_fractionDigits, opt_template,
```
Comment (ger-benjamin) [inline src/services/filters.js]: Remove the "opt_localize" params

File: test/spec/services/filters.spec.js [hunk 2]
```diff
@@ -58,8 +58,6 @@ describe('ngeo.Filters', function() {
           '2,600,000, 1,600,000');
       expect(ngeoNumberCoordinates(co, 0, '{x}, {y}', true)).toBe(
```
Comment (ger-benjamin) [inline test/spec/services/filters.spec.js]: Remove also this function

**Comment by fredj** [comment]
LGTM

**Comment by sbrunner** [comment]
Thanks :-)

**Pull request merged**